rails-commenting.rb

```diff
@@ -7,24 +7,31 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+#The first step is to create a controller method that will access all the items from the database.
```

Comment:
Close thats what the index method does, THis here is a class of BlogPostsController which is a child of ApplicationController